validation/src/test/scala/hmda/validation/rules/lar/validity/V545Spec.scala

```diff
@@ -0,0 +1,36 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V545Spec extends LarEditCheckSpec with BadValueUtils {
+
+  property("Valid when HOEPA status is 2") {
+    forAll(larGen) { lar =>
+      whenever(lar.hoepaStatus == 2) {
+        lar.mustPass
+      }
+    }
+  }
+
+  property("Valid when Lien status != 3") {
+    forAll(larGen) { lar =>
+      whenever(lar.lienStatus != 3) {
+        lar.mustPass
+      }
+    }
+  }
+
+  val invalidHoepaGen: Gen[Int] = Gen.choose(Int.MinValue, Int.MaxValue).filter(_ != 2)
```

Comment:
Oops, correction:  It will return `Option[None]` if it happens to pick 2.  

I still think that `retryUntil()` would be a more efficient option in this case (since there's such a small chance of it equalling 2).